docs/how-to/use-flask-extension.rst

```diff
@@ -32,6 +32,9 @@ Managing project files with the flask extension
 The prime declaration must be included in the specially-named
 ``flask/install-app`` section to instruct the flask extension on which files
 to include or exclude from the project directory in the ROCK image.
+And since the extension places the files from the project folder in the
```

Comment:
Hmm a better wording might be:

```
The extension places the files from the project folder in the srv/flask/app`
directory in the final image - therefore, all inclusions and exclusions must
be prefixed with `src/flask/app`.

Reply:
Updated, thank you! I have also updated the extension based on last week's meeting, changing the base directory to `/flask`, and mandating the `app:app` WSGI path.